redbot/cogs/modlog/modlog.py

```diff
@@ -97,17 +97,24 @@ def __init__(self, bot: Red):
         else:
             await ctx.send(embed=await case.get_case_msg_content())
 
-    @commands.command()
+    @commands.command(usage="[case] <reason>")
     @commands.guild_only()
-    async def reason(self, ctx: commands.Context, case: int, *, reason: str = ""):
+    async def reason(self, ctx: commands.Context, *, reason: str):
         """Lets you specify a reason for mod-log's cases
+        
         Please note that you can only edit cases you are
-        the owner of unless you are a mod/admin or the server owner"""
+        the owner of unless you are a mod/admin or the server owner.
+        
+        If no number is specified, the latest case will be used."""
         author = ctx.author
         guild = ctx.guild
-        if not reason:
-            await ctx.send_help()
-            return
+        potential_case = reason.split()[0]
+        if potential_case.isnumeric():
```

Comment:
This should be using `isdigit()` not `isnumeric()`. The later of these two is used when you need to make extra considerations for unicode numeric-type values like super/sub script, ½, etc.

Reply:
Done 👌